Tests/MapboxMapsTests/Style/AttributionTests.swift

```diff
@@ -0,0 +1,161 @@
+import XCTest
+@testable import MapboxMaps
+import Foundation
+import UIKit
+
+class AttributionTests: XCTestCase {
+
+    let sdkVersion = Bundle.mapboxMapsMetadata.version
+
+    override func setUp() {
+        super.setUp()
+        Bundle.mapboxMapsMetadata.version = "AttributionTests"
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        Bundle.mapboxMapsMetadata.version = sdkVersion
+    }
```

Comment:
I was surprised that `mapboxMapsMetadata` was settable. That looks like a bug in my opinion. Could you avoid setting it here and use string interpolation to create the expected value below?